include/swift/AST/DiagnosticsParse.def

```diff
@@ -866,6 +866,17 @@ ERROR(initializer_as_typed_pattern,none,
 ERROR(unlabeled_parameter_following_variadic_parameter,none,
       "a parameter following a variadic parameter requires a label", ())
 
+ERROR(enum_element_empty_arglist,none,
+      "enum element with associated values must have at least one "
+      "associated value", ())
+WARNING(enum_element_empty_arglist_swift4,none,
+        "enum element with associated values must have at least one "
+        "associated value; this will be an error in Swift 5", ())
```

Comment:
we don't say "this will be an error in Swift X" in other similar warnings, maybe remove that part

Reply:
`enum_element_empty_arglist_swift4` mentions "Swift 5", but I agree.  "In the future" works just as well.